pkgs/by-name/dn/dnsjit/package.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, makeWrapper
+, autoreconfHook
+, autoconf
+, automake
+, libpcap
+, luajit
+, lmdb
+, libck
+, gnutls
+, lz4
+}:
```

Comment:
autoconf, automake and makeWrapper are all not used so you should remove them. You can install nil as a lsp and it will tell you if something isn't used and some other things.